src/components/Main.js

```diff
@@ -2,17 +2,164 @@ import { Component } from 'react';
+          "title": "Markhor",
+          "description": "These wild goats eat snakes, then secrete a foam that locals fight over for the antivemon properties -- how cool is that?",
+          "keyword": "markhor",
+          "horns": 2
+        },
+        {
+          "image_url": "http://www.zooborns.com/.a/6a010535647bf3970b0223c84d5959200c-800wi",
+          "title": "Baby markhor",
+          "description": "Even the babies are adorable",
+          "keyword": "markhor",
+          "horns": 2
+        },
+        {
+          "image_url": "https://images.unsplash.com/photo-1558560063-931ca9822a0c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=634&q=80",
+          "title": "Mouflon",
+          "description": "Those horns though",
+          "keyword": "mouflon",
+          "horns": 2
+        },
+        {
+          "image_url": "https://images.unsplash.com/photo-1556890077-020ec300d5db?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1353&q=80",
+          "title": "Addax",
+          "description": "This guy is basically extinct but survives well in captivity, so they're frequently found in zoos",
+          "keyword": "addax",
+          "horns": 2
+        },
+        {
+          "image_url": "https://cbsnews3.cbsistatic.com/hub/i/r/2013/03/05/5b414225-a645-11e2-a3f0-029118418759/thumbnail/620x350/2d4cf24685b45c22912e64d2004fec8d/Baby_Mouflon_Wild_Sheep.jpg",
+          "title": "Baby mouflon",
+          "description": "The cuteness that is a baby mouflon asleep",
+          "keyword": "mouflon",
+          "horns": 2
+        },
+        {
+          "image_url": "https://images.unsplash.com/photo-1514036783265-fba9577fc473?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80",
+          "title": "Happy Jackson's Chameleon",
+          "description": "These are really common in Hawaii",
+          "keyword": "chameleon",
+          "horns": 2
+        },
+        {
+          "image_url": "https://imgc.allpostersimages.com/img/print/posters/dlillc-jackson-s-chameleon_a-G-13448768-14258384.jpg",
+          "title": "Serious Jackson's Chameleon",
+          "description": "This one is very serious.",
+          "keyword": "chameleon",
+          "horns": 3
+        },
+        {
+          "image_url": "https://www.nps.gov/band/learn/nature/images/short-horned-lizard-open-mouth-18.jpg?maxwidth=650&autorotate=false",
+          "title": "Horned Lizard",
+          "description": "Fave food: ants",
+          "keyword": "lizard",
+          "horns": 100
+        },
+        {
+          "image_url": "https://upload.wikimedia.org/wikipedia/commons/thumb/d/df/Smaug_par_David_Demaret.jpg/290px-Smaug_par_David_Demaret.jpg",
+          "title": "Smaug",
+          "description": "Fan illustration of Smaug from 'The Hobbit'",
+          "keyword": "dragon",
+          "horns": 100
+        }
+      ]
+    };
+  }
```

Comment:
Good work so far!

- I would check with Sara on whether you want the json data to be stored in state. 
- Please create a separate JSON file and then import the json data into the component. Grab a TA to help you with this if you have any questions. I am also happy to help! 👍